apps/nuxt3-ssr/pages/[schema]/ssr-catalogue/[resourceType]/[resource]/index.vue

```diff
@@ -0,0 +1,181 @@
+<script setup lang="ts">
+import { LocationQueryValue } from ".nuxt/vue-router";
+import { gql } from "graphql-request";
+import {
+  IColumn,
+  ISection,
+  ISchemaMetaData,
+  ITableMetaData,
+} from "interfaces/types";
+const config = useRuntimeConfig();
+const route = useRoute();
+const resourceName: string = route.params.resourceType as string;
+const schemaName = route.params.schema.toString();
+const metadata = await fetchMetadata(schemaName);
+
+const tableMetaDataFinderResult = metadata.tables.find(
+  (t: ITableMetaData) =>
+    t.id.toLocaleLowerCase() === resourceName.toLocaleLowerCase()
+);
+
+const tableMetaData = computed(() => {
+  if (tableMetaDataFinderResult) {
+    return tableMetaDataFinderResult;
+  } else {
+    throw new Error(`Table metadata not found for ${resourceName}`);
+  }
+});
+const resourceType = tableMetaData.value.id;
+const externalSchemaNames: string[] = extractExternalSchemas(metadata);
+const externalSchemas = await Promise.all(
+  externalSchemaNames.map(fetchMetadata)
+);
+const schemas = externalSchemas.reduce(
+  (acc: Record<string, ISchemaMetaData>, schema) => {
+    acc[schema.name] = schema;
+    return acc;
+  },
+  { [schemaName]: metadata }
+);
+
+const fields = buildRecordDetailsQueryFields(schemas, schemaName, resourceType);
+
+const keys = route.query.keys;
+const keysObject = locationQueryValueToObject(keys);
+const filter = buildFilterFromKeysObject(keysObject);
+
+const query = gql`
+  query ${resourceType}($filter: ${resourceType}Filter) {
+    ${resourceType}( filter: $filter  ) {
+      ${fields}
+    }
+  }`;
+
+let resource: any;
+let sections: any;
+let tocItems: { label: string; id: string }[];
+
+const { data, pending, error, refresh } = await useFetch(
+  `/${route.params.schema}/catalogue/graphql`,
+  {
+    baseURL: config.public.apiBase,
+    method: "POST",
+    body: { query, variables: { filter: filter } },
+  }
+);
+
+console.log(query);
```

Comment:
remove

Reply:
suggest to leave it ( for now ) its nice to be able to see the constructed query ( fetch from node side does not showup in browser)

Reply:
agree.